docs/user/service-catalog.md

```diff
@@ -0,0 +1,275 @@
+
+WARN  ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.
+nami    INFO  Initializing apache
+apache  INFO  ==> Patching httpoxy...
+apache  INFO  ==> Configuring dummy certificates...
+nami    INFO  apache successfully initialized
+nami    INFO  Initializing php
+nami    INFO  php successfully initialized
+nami    INFO  Initializing mysql-client
+nami    INFO  mysql-client successfully initialized
+nami    INFO  Initializing libphp
+nami    INFO  libphp successfully initialized
+nami    INFO  Initializing wordpress
+mysql-c INFO  Trying to connect to MySQL server
+mysql-c INFO  Found MySQL server listening at 93489418-7a54-4b2c-b807-d239cb26ad5d.mysql.database.azure.com:3306
+mysql-c INFO  MySQL server listening and working at 93489418-7a54-4b2c-b807-d239cb26ad5d.mysql.database.azure.com:3306
+```
+
+## Example 2. Chartmuseum using a GCP storage bucket
+
+[ChartMuseum](https://github.com/helm/chartmuseum) is an open-source project
+that allows you to easily have your own valid Helm Chart Repository. It has
+support for different cloud storage backends, including [Google Cloud Storage](https://cloud.google.com/storage/).
+
+In this example we will create a Google Cloud Storage instance with Kubeapps
+Service Catalog integration, and we will use it as backend for a ChartMuseum
+deployment.
+
+### Provisioning the Google Cloud Storage bucket.
+
+Under `Service Instances` click on `Deploy Service Instance` and select Google
+Cloud Storage:
+
+![Google Cloud Storage](../img/gcp-storage.png)
+
+Select the `Beta plan` and provide a name. 
+
+We will be adding some parameters to configure the name of the bucket and
+the location. We will leave the rest of the options with the default values:  
+
+![Google Cloud Storage options](../img/gcp-storage-options.png)
+
+Once provisioned, we will add a binding clicking on `Add binding`:
+
+![Google Cloud Storage binding options](../img/gcp-storage-binding-options.png)
+
+That will create a new service account in your GCP project with the right
+persmissions to access the bucket.
+
+Once the binding creation has been completed, the details of the created
+secret can be explored directly from the same page, clicking on `show`.
+
+It is important to understand the schema of the secret, as it is dependent
+on the broker and the instance. For Google Storage the secret will have the
+following schema:
+
+```
+bucketId: the id of the bucket
+privateKeyData: JSON with authantication details for the bucket
+projectId: the GCP project where the bucket was created
+serviceAccount: the service account to access the bucket
+```
+
+### Deploy Chartmuseum
+
+We will now deploy ChartMuseum, using Google Cloud Storage as backend.
+In the `Catalog` we will search for `chartmuseum`:
+
+![Searching ChartMuseum](../img/search-chartmuseum.png)
+
+Click on `Deploy` and modify the following values:
+
+```
+env.open.STORAGE: google
```

Comment:
maybe it's better to copy here a YAML snippet that users can paste in Kubeapps directly? (instead of looking for the specific fields)

Reply:
mmm, I am not sure. The problem is that in Kubeapps, the default behaviour (I believe) is to go through the values.yaml and replace the values you want to replace, rather than replacing the whole values.yaml. Those values are in very different places, so adding the full values.yaml would be too big. What do you think?

Reply:
Yes, what I meant is, to avoid having to navigate through the whole values.yaml, tell the user something like: 

```
Replace the default values.yaml with the following snippet and fill it with the values from the service binding:

env:
  open:
    STORAGE: google
    STORAGE_GOOGLE_BUCKET: <name of the created bucket in the instance>
...
```

There is no need to specify the whole values.yaml, specifying just those fields will have the same effect.